packages/modules/promotion/src/models/campaign.ts

```diff
@@ -40,8 +39,8 @@ export default class Campaign {
   @Property({ columnType: "text", nullable: true })
   description: string | null = null
 
-  @Property({ columnType: "text", nullable: true })
-  currency: string | null = null
+  @Property({ columnType: "text" })
+  currency: string
```

Comment:
question: If the campaign has a spend-budget, I would imagine the currency could be nullable. What prompted this change?